wger/manager/templates/schedule/view.html

```diff
@@ -85,7 +85,9 @@
         <td>
             {% if schedule.is_loop %}
                 <span class="{% fa_class 'check' %}"></span>
-                <small>{% trans "This schedule is a loop and will go through the workouts above till you deactivate it" %}</small>
+                <small>
```

Comment:
I'm not really sure if this HTML file and some other that you have made changes to above required to have the flake8 changes. In my view, I thought it only applied to `.py` files